provider/job_utils.py

```diff
@@ -244,6 +244,9 @@ def is_block_job_running(vm, jobid, tmo=200):
         if not job:
             LOG_JOB.warn('job %s cancelled unexpectedly', jobid)
             break
+        elif job['status'] != "running":
+            LOG_JOB.warn('job %s is not in running status', jobid)
```

Comment:
What about use LOG_JOB.debug instead of warn, in case we get a Test Warn exception in the end.

Reply:
That's a good idea, will updated it later.